### PR TITLE
feat(amplify-flutter) project setup section

### PR DESCRIPTION
**Pull request by haverchuck**
Adds Project Setup section for flutter, with small changes for android/ios to account for differing CLI installations.

### CODE REVIEW

File: docs/lib/project-setup/fragments/flutter/create-application/10_createProject.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+![](~/images/lib/getting-started/flutter/set-up-android-studio-welcome.png)
+
+ In **Select a Project Template**, select **Flutter Application**. Press **Next**.
+
+Next, configure your project:
+
+1. Enter *MyAmplifyApp* in the **Name** field
+2. Make sure your Flutter SDK path is set correctly to where it is installed on your machine 
+3. Press **Next**.  On the next screen, press **Finish**.
```
Comment (sammartinez) [inline docs/lib/project-setup/fragments/flutter/create-application/10_createProject.md]:
```suggestion
3. Press **Next**. On the next screen, press **Finish**. 
```

File: docs/lib/project-setup/fragments/flutter/create-application/30_provisionBackend.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+amplify init
+```
+
+Enter the following when prompted:
+
+```console
+? Enter a name for the environment
+    `dev`
+? Choose your default editor:
+    `IntelliJ IDEA`
```
Comment (sammartinez) [inline docs/lib/project-setup/fragments/flutter/create-application/30_provisionBackend.md:13]: Is Android studio need to be set here?

**Review by sammartinez — COMMENTED** [review]
Couple of callouts

File: docs/lib/project-setup/fragments/flutter/create-application/30_provisionBackend.md (hunk 2 above)
Comment (haverchuck) [reply docs/lib/project-setup/fragments/flutter/create-application/30_provisionBackend.md:13]: I believe the two are synonymous - AS is built on IntilliJ.

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Pull request merged**